packages/lab/src/pill-next/PillNextCloseButton.css

```diff
@@ -0,0 +1,61 @@
+.saltPillNextCloseButton {
+  appearance: none;
+  -webkit-appearance: none;
+  display: inline-flex;
+  align-items: center;
+  background: var(--salt-actionable-primary-background);
+  border-radius: 0;
+  border: 0;
+  height: calc(var(--salt-size-base) - var(--salt-spacing-100));
+  min-height: var(--salt-text-minHeight);
+  outline: 0;
+  position: relative;
+  gap: var(--salt-spacing-50);
+  padding-left: var(--salt-spacing-50);
+  padding-right: var(--salt-spacing-50);
+  color: var(--salt-actionable-primary-foreground);
+  font-family: var(--salt-text-fontFamily);
+  font-size: var(--salt-text-fontSize);
+  font-weight: var(--salt-text-fontWeight);
+  line-height: var(--salt-text-label-lineHeight);
+  cursor: var(--salt-selectable-cursor-hover);
+}
+
+.saltPillNextCloseButton .saltIcon {
+  --saltIcon-color: currentColor;
+}
+
+.saltPillNextCloseButton:hover {
+  color: var(--salt-actionable-primary-foreground-hover);
+  background: var(--salt-actionable-primary-background-hover);
+}
+
+.saltPillNextCloseButton.saltPillNextCloseButton-disabled:hover {
+  color: var(--salt-actionable-primary-foreground);
+  background: var(--salt-actionable-primary-background);
+}
+
+.saltPillNextCloseButton.saltPillNextCloseButton-active,
+.saltPillNextCloseButton.saltPillNextCloseButton:active {
+  background: var(--salt-actionable-primary-background-active);
+  color: var(--salt-actionable-primary-foreground-active);
+}
+
+.saltPillNextCloseButton.saltPillNextCloseButton-disabled.saltPillNextCloseButton-active,
+.saltPillNextCloseButton.saltPillNextCloseButton-disabled:active {
```

Comment:
how do we get into a disabled active state?

Reply:
so is this necessary?

Reply:
removed :)